package.json

```diff
@@ -9,7 +9,7 @@
     "start:dev": "nodemon --exec babel-node server/index.js",
     "heroku-postbuild": "npm run db:rollback && npm run build",
     "debug": "nodemon --exec babel-node server/index.js --inspect",
-    "test": "npm run db:roll && npm run db:seed && nyc mocha --recursive test --timeout 10000 -r esm --exit && npm run db:back",
+    "test": "npm run db:roll && npm run db:seed && nyc mocha --recursive test --timeout 20000 -r esm --exit && npm run db:back",
```

Comment:
why did you change this to 20000

Reply:
the test was failing due to timeouts, had to extend timeouts